ext/DimensionalDataMakie.jl

```diff
@@ -432,8 +432,13 @@ end
     # These can just forward to the relevant converts.
     Makie.expand_dimensions(t::Makie.PointBased, A::AbstractDimVector) = Makie.convert_arguments(t, A)
     Makie.expand_dimensions(t::Makie.PointBased, A::AbstractDimMatrix) = Makie.convert_arguments(t, A)
-    Makie.expand_dimensions(t::SurfaceLikeCompat, A::AbstractDimMatrix) = Makie.convert_arguments(t, A)
-    Makie.expand_dimensions(t::Makie.CellGrid, A::AbstractDimMatrix) = Makie.convert_arguments(t, A)
```

Comment:
as I tested on https://github.com/rafaqz/Rasters.jl/pull/723, it looks like this is not enough to cover the uses cases there.

Reply:
No, that's a different issue - the conversions for vertexgrid have to be different than those for cellgrid, and I don't think those are implemented in here yet.  Will look into it.

Reply:
Can we just delete everything in Rasters? It's an AbstractDimArray anyway...

Reply:
Yeah I guess the code is pretty old :D 

The only thing that's really needed is plot I think

Reply:
Ah shoot, we can't drop Rasters' Makie because it has the methods to populate missings and stuff.  It's already pretty generic though and everything _should_ be forwarded to DimensionalData...

Reply:
Ohh right, makes sense. Can we just replace missings there and rewrap as `DimArray`? We will still need all the same dispatch methods to avoid ambiguities.

We also need the dispatch for handling rotated projections and other weird things (or maybe GeoMakie can deal with that?)

Reply:
GeoMakie can deal with that to an extent but if you're plotting on a regular axis you still need that.

Reply:
Yeah, but maybe we need a GeoMakie extension to just plot rotations and similar directly rather than resampling them first

Reply:
We could!  It'd add a lot more complexity though, and I wouldn't trust that with e.g. 2d to 3d transformations.